warp10/src/main/java/io/warp10/continuum/store/Directory.java

```diff
@@ -1851,6 +1867,9 @@ public void run() {
               // 128bits
               if (null == directory.metadatas.get(metadata.getName()).put(labelsId, metadata)) {
                 Sensision.update(SensisionConstants.SENSISION_CLASS_CONTINUUM_DIRECTORY_GTS, Sensision.EMPTY_LABELS, 1);
+                if (!tracked) {
```

Comment:
Is this case actually plausible? How can tracked be true if the metadata were not known?

Reply:
True, I can remove the tracked variable in this case